run.ps1

```diff
@@ -124,10 +124,10 @@ function Get-KoreBuild {
             }
 			 
             # Hack to use Core version
-            Write-Host "!!! Hack to use .NET Core SDK 5.0.301"
```

Comment:
update `run.sh` as well for Linux CI build.

Reply:
fixed